File: src/main/kotlin/com/stannis/dataModel/complexStatementTypes/FunctionCallWithDeclaration.kt
package com.stannis.dataModel.complexStatementTypes

import com.stannis.dataModel.BodyParent
import com.stannis.dataModel.Statement
import com.stannis.dataModel.statementTypes.FunctionCalls

data class FunctionCallWithDeclaration(
    override val `$type`: String? = "FunctionCallWithDeclaration",
    var functionCalls: FunctionCalls?,
    var declaration: Statement?,
    var complexClass: ComplexCompositeTypeSpecifier?
) : Statement, BodyParent

File: Interpreter/Interpreter/Models/serialize/complexStatementTypes/FunctionCallWithDeclaration.cs
namespace Interpreter.Models.serialize.complexStatementTypes
{

    public class FunctionCallWithDeclaration : IStatement
    {
        public IStatement functionCalls { set; get; }
        
        public IStatement declaration { set; get; }
        
        public ComplexCompositeTypeSpecifier complexClass { set; get; }
        
    }

}